torch/nn/modules/loss.py

```diff
@@ -297,6 +297,38 @@ def forward(self, log_input: Tensor, target: Tensor) -> Tensor:
                                   eps=self.eps, reduction=self.reduction)
 
 
+class GaussianNLLLoss(_Loss):
+    r"""Negative log likelihood loss with Gaussian distribution
+
+    The loss can be described as::
+
+    target ~ N(input, var)
+
+    loss(input, target, var) = %TODO - need help with math
```

Comment:
You can do something similar to the kldiv below:
```
    .. math::
        l(x,y) = L = \{ l_1,\dots,l_N \}, \quad
        l_n = y_n \cdot \left( \log y_n - x_n \right)
```
The syntax inside the `.. math::` construct is latex formula syntax IIRC